### PR TITLE
[Doc] Update FE_configuration.md

### PR DESCRIPTION
**Pull request by nshangyiming**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/zh/administration/FE_configuration.md [hunk 1]
```diff
@@ -530,6 +530,17 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
...
 - 默认值：200 \* 1024 \* 1024 \* 1024
 
+#### storage_usage_hard_limit_percent
+
+- 含义：如果 BE 存储目录空间使用率超过该值且剩余空间小于 `storage_usage_hard_limit_reserve_bytes`，会拒绝 Load 和 Restore 作业。需要同步修改 BE 配置 `storage_flood_stage_usage_percent`。
+- 默认值：95
+
+#### storage_usage_hard_limit_reserve_bytes
+
+- 含义：默认 100 GB，单位为 Byte，如果 BE 存储目录下剩余空间小于该值且空间使用率超过 `storage_usage_hard_limit_percent`，会拒绝 Load 和 Restore 作业。需要同步修改 BE 配置 `storage_flood_stage_left_capacity_bytes`。
```
Comment (evelynzhaojie) [inline docs/zh/administration/FE_configuration.md:540]: "需要同步修改 " --- 这句话没太看懂，”同步修改“ 是和谁同步一起修改，是说在这种剩余空间不足的情况下，需要 xxx。还是说 storage_usage_hard_limit_reserve_bytes 这个参数单独设置不生效，需要和 BE 参数`storage_flood_stage_left_capacity_bytes` 一起配置？

Comment (EsoragotoSpirit) [reply docs/zh/administration/FE_configuration.md:540]: @nshangyiming

File: docs/zh/administration/FE_configuration.md [hunk 2]
```diff
@@ -530,6 +530,17 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - 单位：字节
 - 默认值：200 \* 1024 \* 1024 \* 1024
 
+#### storage_usage_hard_limit_percent
+
+- 含义：如果 BE 存储目录空间使用率超过该值且剩余空间小于 `storage_usage_hard_limit_reserve_bytes`，会拒绝 Load 和 Restore 作业。需要同步修改 BE 配置 `storage_flood_stage_usage_percent`。
```
Comment (evelynzhaojie) [inline docs/zh/administration/FE_configuration.md:535]: 先说明这个参数的含义是什么，再写 ”如果xxx“

File: docs/zh/administration/FE_configuration.md (hunk 1 above)
Comment (evelynzhaojie) [inline docs/zh/administration/FE_configuration.md:540]: 含义：BE 存储目录剩余空间。如果 xxx

Comment (nshangyiming) [reply docs/zh/administration/FE_configuration.md:540]: 是的

File: docs/zh/administration/FE_configuration.md (hunk 2 above)
Comment (nshangyiming) [reply docs/zh/administration/FE_configuration.md]: 含义：BE 存储目录整体磁盘空间使用率的硬限制阈值。

File: docs/zh/administration/FE_configuration.md (hunk 1 above)
Comment (nshangyiming) [reply docs/zh/administration/FE_configuration.md]: 含义：BE 某存储目录磁盘剩余空间的硬限制阈值。

**Review by EsoragotoSpirit — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**